pyproject.toml

```diff
@@ -0,0 +1,88 @@
+[tool.poetry]
+name = "spikeinterface"
+version = "0.94.0.dev1"
+license = "MIT"
+
+authors = ["Cole Hurwitz", "Jeremy Magland", "Alessio Paolo Buccino <alessiop.buccino@gmail.com>", "Matthias Hennig", "Samuel Garcia"]
+maintainers = ["Alessio Paolo Buccino <alessiop.buccino@gmail.com>"]
+
+homepage = "https://github.com/SpikeInterface/spikeinterface"
+description = "Python toolkit for analysis, visualization, and comparison of spike sorting output"
+readme = "README.md"
+
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+
+[tool.poetry.dependencies]
+python = ">=3.7,<3.10"
+
+probeinterface = "^0.2.6"
+
+neo = "^0.10.0"
+joblib = "^1.1.0"
+tqdm = "^4.62.3"
+numpy = "^1.21"
+
+# scipy = {version = "^1.8.0", optional = true}
+scikit-learn = {version = "^1.0.2", optional = true}
+h5py = {version = "^3.6.0", optional = true}
+matplotlib = {version = "^3.5.1", optional = true}
+networkx = {version = "^2.7", python= ">=3.8", optional = true}
+pandas = [
+    {version = "*", python= "~3.7", optional = true},
+    {version = "^1.4.1", python= ">=3.8", optional = true}
+]
+
+datalad = {version = "^0.15.6", optional = true}
```

Comment:
Should I add `datalad` to full? It is mentioned in the `installation.rst`, but is not in `full` nor `extractors`.